spec/models/season/phase_switcher_spec.rb

```diff
@@ -1,35 +1,43 @@
 require 'spec_helper'
 
+describe Season::PhaseSwitcher do
 
-context 'when switching phases' do
-  subject { Season.current }
+  it "is available as described class" do
+    expect(described_class).to eq Season::PhaseSwitcher
+  end
+
+  context 'when switching phases' do
+    season = Season.current
```

Comment:
Outside of examples, it's best practice and way more understandable for every other developer if you'd use `let` here:
```ruby
let(:season) { Season.current }
let!(:season) { Season.current } # if you need to have it called *before* the first call to "season"
```